File: csharp/module1/solutions/solution5.cs
public int GetFireballDamage(int distance, int wizardLevel) {
    const int baseDamage = 20;
    const int distanceMultiplier = 3;
    const int wizardLevelMultiplier = 6;

    int distanceDamage = distance * distanceMultiplier;
    int wizardLevelDamage = wizardLevel * wizardLevelMultiplier;
    return baseDamage + distanceDamage + wizardLevelDamage;
}

File: javascript/module1/solutions/solution5.js
function getFireballDamage(distance, wizardLevel) {
    const baseDamage = 20;
    const distanceMultiplier = 3;
    const wizardLevelMultiplier = 6;
  
    const distanceDamage = distance * distanceMultiplier;
    const wizardLevelDamage = wizardLevel * wizardLevelMultiplier;
    return baseDamage + distanceDamage + wizardLevelDamage;
  }
